### PR TITLE
Add Ethers.get_transaction_receipt

### PR DESCRIPTION
**Pull request by calvin-nl**
Add Ethers.get_transaction_receipt

### CODE REVIEW

File: lib/ethers.ex [hunk 1]
```diff
@@ -153,6 +153,23 @@ defmodule Ethers do
     |> post_process(nil, :get_transaction)
   end
 
+  @doc """
+  Returns the native transaction (ETH) receipt by transaction hash.
```
Comment (alisinabh) [inline lib/ethers.ex:157]: What does `ETH` and native transaction mean in this context?

Can we just say `Returns receipt of a transaction by it's hash`?

File: lib/ethers.ex [hunk 2]
```diff
@@ -153,6 +153,23 @@ defmodule Ethers do
...
+
+  ## Parameters
+  - tx_hash: Transaction hash which the transaction receipt is queried for.
+  - overrides:
+    - rpc_opts: Specific RPC options to specify for this request.
```
Comment (alisinabh) [inline lib/ethers.ex:162]: Seems like I also forgot to put `rpc_client`option in `get_transaction/2` too

```suggestion
  - overrides:
    - rpc_client: The RPC module to use for this request (overrides default).
    - rpc_opts: Specific RPC options to specify for this request.
```

**Review by alisinabh — APPROVED** [review]
Great addition! Thank you @calvin-nl 🚀 

Just two doc related comments and then this is ready to merge.

File: lib/ethers.ex (hunk 1 above)
Comment (calvin-nl) [reply lib/ethers.ex]: Yup, makes sense

File: lib/ethers.ex (hunk 2 above)
Comment (calvin-nl) [reply lib/ethers.ex]: Will do :)

**Comment by alisinabh** [comment]
❤️ 💜 ❤️ 💜

**Pull request merged**